package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "p2p-graph",
   "description": "Real-time P2P network visualization with D3",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
Please leave the version bump to the maintainers. Otherwise, what happens if we merge another PR first and that one becomes 1.1.2? Then the version bump in this PR will be incorrect. Best to not touch the version.